pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "alfalfa-client"
-version = "0.1.1"
+version = "0.2.0"
 description = "A standalone client for the NREL Alfalfa application"
 authors = [
```

Comment:
add your name to the author list?

`Tobias Shapinsky <tobias.shapinksy@nrel.gov>`

Reply:
+1!